vllm/worker/habana_model_runner.py

```diff
@@ -1938,14 +1940,22 @@ def execute_model(
         return [output]
 
     def shutdown_inc(self):
-        print('inc shutdown')
-        if (model_config := getattr(self, "model_config", None)) and \
-                         getattr(model_config, "quantization", None) == 'inc':
-            print('inc shutdown start')
+        from operator import attrgetter
+        can_finalize_inc = False
+        try:
+            is_inc_used = attrgetter("model_config.quantization")(
+                self) == 'inc'
+            is_model_initialized = attrgetter("model.model")(self) is not None
+            is_inc_initialized = attrgetter("inc_initialized_successfully")(
+                self)
+            can_finalize_inc = is_inc_used and is_model_initialized and \
+                is_inc_initialized
+        except AttributeError:
+            pass
+        if can_finalize_inc:
             from neural_compressor.torch.quantization import (
                 finalize_calibration)
             finalize_calibration(self.model.model)
```

Comment:
Oh, I thought attrgetter can handle AttributeErrors ...

So what's the point of using it at the moment? Since we're catching the exception, couldn't we do:
```python
        try:
            is_inc_used = self.model_config.quantization == 'inc'
            is_model_initialized = self.model.model is not None
            is_inc_initialized = self.inc_initialized_successfully
            can_finalize_inc = is_inc_used and is_model_initialized and \
                is_inc_initialized
        except AttributeError:
            pass
```
or even better:
```python
 can_finalize_inc = (self.model_config.quantization == 'inc') and \
    (self.model.model is not None) and \
    self.inc_initialized_succesfully
```

Reply:
fixed + added protection against double shutdown